changelogs/fragments/347-ansible-test-python-requires.yml

```diff
@@ -0,0 +1,2 @@
+trivial:
+  - ci - add test-deps to integration test workflows (https://github.com/ansible-collections/community.digitalocean/issues/342).
```

Comment:
This changelog entry is wrong, but the actual change LGTM. Thanks!

Reply:
> This changelog entry is wrong, but the actual change LGTM. Thanks!

Ah thank you, bad copy/paste, will fix.